honeybee_schema/update/version_1_40_1.py

```diff
@@ -0,0 +1,158 @@
+"""Changes associated with version Honeybee schema version 1.40.1."""
+from ._base import Updater
+
+
+class Version_1_40_1(Updater):
+
+    @staticmethod
+    def update(model_dict):
+        """Implement changes in a Model dict to make it compatible with version 1.40.1.
+        """
+        if 'energy' in model_dict['properties']:
+            if 'hvacs' in model_dict['properties']['energy']:
+                for hvac in model_dict['properties']['energy']['hvacs']:
+                    if hvac['type'] != 'IdealAirSystemAbridged':  # it's detailed
+                        hvac['vintage'] = VINTAGE_MAPPER[hvac['vintage']]
+                        hvac['equipment_type'] = EQUIPMENT_MAPPER[hvac['equipment_type']]
+        return model_dict
```

Comment:
👍 very nice! I like the approach here and starting from the first available update to the last is convincing. I just think we can use these update functions outside of the class to exactly do the same.